constraint/pkg/client/drivers/remote/remote.go

```diff
@@ -90,6 +95,11 @@ func (d *driver) DeleteModule(ctx context.Context, name string) (bool, error) {
 	return err == nil, err
 }
 
+// DeleteModules implements drivers.Driver
```

Comment:
The not implemented errors are causing the tests of the remote driver to fail. We can disable those tests for now, I think.